Sources/Kitura/Router.swift

```diff
@@ -156,6 +164,9 @@ public class Router {
     /// let router = Router()
     /// router.setDefault(templateEngine: MyTemplateEngine())
     /// ```
+    /// If the template engine doesn't provide a default extension you can provide one using
+    /// `add(templateEngine:forFileExtensions:useDefaultFileExtension:)`. If a router instance doesn't
+    /// have a template engine registered that can render the given template file an error is thrown.
```

Comment:
Could you add in what the error is, then people can recognise that it's the "template engine missing" error and not something else.